napari/_qt/layers/tests/test_qt_image_base_layer_.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import pytest
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import QPushButton
+
+from ....layers import Image, Surface
```

Comment:
I feel like in our tests we have tended to use absolute imports. I'm not quite sure why, but I probably wouldn't change convention here